WcaOnRails/app/models/competition.rb

```diff
@@ -489,7 +490,11 @@ def info_for(user)
       info[:upload_results] = I18n.t('competitions.messages.upload_results')
     end
     if self.in_progress? && !self.cancelled?
-      info[:in_progress] = I18n.t('competitions.messages.in_progress', date: I18n.l(self.end_date, format: :long))
+      if self.use_wca_live_for_scoretaking
```

Comment:
Now with your additional input about using `NULL` as default in the DB, I have to reconsider my previous review comment about accessing the field.

I think with the changes as they are now, it would be fine to create a separate method like
```ruby
def using_wca_live?
  # Comment that explains why we explicity check for `true`
  self.use_wca_live_for_scoretaking == true
end
```

and use it. The point is that `if self.use_wca_live_for_scoretaking` evaluates as "false" both for the actual `false` in the DB as well as for `null`. In this specific case, it would still lead to the desired behavior "accidentally", but I'm always a friend of spelling things out explicitly instead of (ab)using programming languages' default behavior (like evaluating `null` as `false`-ish in an `if` context for example).

The comment should also spell out that we explicitly chose to allow NULL for historic reasons, because we cannot be sure whether or not some past competition used WCA Live. It can be very similar to the explanation you gave to me in the comment under the migration file below.

Reply:
Resolving this as we're not using NULL as default any longer